src/data/playground-configs.ts

```
@@ -4342,6 +4342,20 @@ export const chartConfigs: Record<string, ChartConfig> = {
+      name: 'General',
+      fields: [
+        {
+          label: 'Image Tag',
+          key: 'image.tag',
+          type: 'text',
+          default: 'v0.0.64',
+          description: 'Pinned Kubernetes MCP Server image tag',
+        },
+      ],
+    },
+  ],
```

Biome (2.5.3)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)